### DISCUSSION TITLE
Support ZeST

### DISCUSSION THREAD

**Discussion by xiaohu2015** [cubiq/ComfyUI_IPAdapter_plus#442] [General]
https://ttchengab.github.io/zest/, the result looks good, maybe can be implemented in this repo @cubiq

**Comment by cubiq** [comment]
This can be already done. It's style transfer + depth map + inpaint mask (as far as I understand)

**Comment by cubiq** [comment]
yeah it works already :smile: 

![gold_strawberry](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/427614/9e6205f4-db22-48e8-8cb0-a47dbd0d1168)

**Comment by GalDude33** [comment]
Can u share the workflow plz? 🙏

**Reply by cubiq** [reply]
no problem


[targeted_style_transfer.json](https://github.com/cubiq/ComfyUI_IPAdapter_plus/files/14931549/targeted_style_transfer.json)
![faceted_gold](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/427614/2349bcfb-3cb4-495b-be33-e1d5af190b90)
![strawberry](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/427614/1c9eb1e4-b50d-4802-89e1-025e13794fd8)
![ComfyUI_temp_pepsm_00100_](https://github.com/cubiq/ComfyUI_IPAdapter_plus/assets/427614/2244b0c6-877f-4480-b42f-161040c7222d)
